### DISCUSSION TITLE
Consider add search result counts for tha "Languages" and "Repositories" panels

### DISCUSSION THREAD

**Discussion by Dentrax** [community/community#9718] [Other Feature Feedback, Questions, & Ideas] [answered]
It would be nice UX.

![Screen Shot 2022-01-06 at 21 37 56](https://user-images.githubusercontent.com/16493751/148433651-1fdddcd8-d213-4959-924a-63d3cf030b4b.png)

**Comment by look** [comment, marked as answer]
Thanks for the suggestion! This is on our roadmap.
